### DISCUSSION TITLE
Wrong Credentials to autheticate via API from YetiForce Portal to CRM

### DISCUSSION THREAD

**Discussion by kilian099** [YetiForceCompany/YetiForceCRM#17457] [How to]
# 🐞 bug report

### ⁉️ Describe the bug

I have been provided with the right credentials, as the tutorial explains, and it is still telling me that the credentials are wrong.

**Comment by rskrzypczak** [comment]
Check if you have an active  `$encryptDataTransfer` in `config\Config.php` (Poratl), if so, deactivate it
